src/Altinn.AccessManagement/Controllers/DelegationsController.cs

```diff
@@ -235,6 +235,7 @@ public async Task<ActionResult> DeletePolicy([FromBody] RequestToDeletePolicyLis
         /// <response code="500">Internal Server Error</response>
         [HttpGet]
         [Authorize]
+        [Authorize(Policy = AuthzConstants.POLICY_MASKINPORTEN_DELEGATION_READ)]
```

Comment:
Read access should also be added to endpoint for getting "inbound" maskinporten delegations